WineCellar/WineCellar.GUI/RegisterWine.xaml.cs

```diff
@@ -130,34 +121,54 @@ private bool isYear(String toValidate)
                 return false;
             }
 
-            if(iyear < 1000 || iyear > 2100)
+            if (iyear < 1000 || iyear > 2100)
             {
                 return false;
             }
 
             return true;
         }
 
-        
+
         private void AttemptRegister(object sender, RoutedEventArgs e)
         {
             bool validate = this.Validation();
             if (validate)
             {
+
+                WineData wine = new WineData();
+
+                wine.Name = name.Text;
+                wine.Age = Convert.ToInt32(year.Text);
+                wine.OriginCountry = country.Text;
+                wine.Stock = 0;
+                wine.Contents = Convert.ToInt32(contents.Text);
+                wine.BuyPrice = Convert.ToDouble(buy.Text);
+                wine.SellPrice = Convert.ToDouble(sell.Text);
+                wine.Alcohol = Convert.ToDecimal(alcohol.Text);
+                wine.Picture = FileContent;
+                wine.TypeId = 0;
+                wine.Description = description.Text;
+                wine.Rating = 5;
+
+                Controller.Data.Create(wine);
+                MainWindow window = new MainWindow();   
```

Comment:
Wellicht van dit scherm een dialog maken, zodat deze over de huidige window staat. Hierdoor hoef je niet telkens de mainwindow opnieuw te maken en blijft deze op de achtergrond. Is te overleggen.

Reply:
Ja, zo stond het niet in het FO/TO. Misschien iets voor de volgende sprint, anders kunnen we dit niet presenteren. Gaat teveel tijd kosten denk ik.